models/source.js

```diff
@@ -21,7 +21,8 @@ var sourceSchema = new mongoose.Schema({
   enabled: {type: Boolean, default: true},
   events: {type: Array, default: []},
   unreadErrorCount: {type: Number, default: 0},
-  lastReportDate: Date
+  lastReportDate: Date,
+  user: {type: mongoose.Schema.Types.ObjectId, ref: 'User', required: false},
```

Comment:
Isn't `required: false` the default?

Also: in other models the type for foreign keys like this is `String`. Should they eventually be refactored? What is the benefit of this style?